src/pages/shared/Modals/RegisterModal/ValidateInputs.js

```diff
@@ -18,7 +18,7 @@ function ValidateRegisterInput(formValues){
         return { success: false, message: 'Пароль має містити більше 8 символів'};
     }
 
-    if (formValues['password'] !== formValues['submit_password']){
+    if (formValues['password'] !== formValues['a']){
```

Comment:
The key thing is not clearly written down
Here is a question from the DEV: "What is a?"